src/vs/workbench/contrib/files/browser/files.ts

```diff
@@ -137,6 +137,15 @@ export function getMultiSelectedResources(resource: URI | object | undefined, li
 		}
 	}
 
+	const activeGroup = editorGroupService.activeGroup;
```

Comment:
I am not sure about this change, we have this method to check for selection in the explorer as far as I can remember, and now we go to the active editor group? I would be very careful here, this sounds like it will break a lot of things...

Reply:
Explorer gets context in a different way: https://github.com/microsoft/vscode/blob/9b37afed101c5340b03c1bca862c1d44f35ec5ad/src/vs/workbench/contrib/files/browser/views/explorerView.ts#L96